app/models/order_item.rb

```diff
@@ -40,7 +40,7 @@ def set_defaults
   end
 
   def update_message(level, message)
-    progress_messages << ProgressMessage.new(:level => level, :message => message)
+    progress_messages << ProgressMessage.new(:level => level, :message => message, :tenant_id => self.tenant_id)
```

Comment:
We should probably have a test validating this so we make sure it isn't missed moving forward.